x-pack/test/apm_api_integration/tests/transactions/transactions_groups_comparison_statistics.ts

```diff
@@ -70,19 +71,19 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         expect(response.status).to.be(200);
 
-        const transactionsGroupsComparisonStatistics = response.body as TransactionsGroupsComparisonStatistics;
+        const {
+          currentPeriod,
+          previousPeriod,
+        } = response.body as TransactionsGroupsComparisonStatistics;
 
-        expect(Object.keys(transactionsGroupsComparisonStatistics).length).to.be.eql(
-          transactionNames.length
-        );
+        expect(Object.keys(currentPeriod).length).to.be.eql(transactionNames.length);
+        expect(Object.keys(previousPeriod).length).to.be.eql(0);
 
         transactionNames.map((transactionName) => {
-          expect(transactionsGroupsComparisonStatistics[transactionName]).not.to.be.empty();
+          expect(currentPeriod[transactionName]).not.to.be.empty();
         });
 
-        const { latency, throughput, errorRate, impact } = transactionsGroupsComparisonStatistics[
-          transactionNames[0]
-        ];
+        const { latency, throughput, errorRate, impact } = currentPeriod[transactionNames[0]];
```

Comment:
Like I said before, I don't think it's great that we are dealing with a key-value map here. What about converting the response to an array just for the test?